tests/scripts/thread-cert/backbone/bbr_5_11_01.py

```diff
@@ -100,7 +100,7 @@ def test(self):
         self.simulator.go(5)
         self.assertEqual('router', self.nodes[BR_1].get_state())
         self.nodes[BR_1].enable_backbone_router()
```

Comment:
Unrelated to this PR, but it may not necessary to `enable_backbone_router()` explicitly for first PBBR now given that it is enabled by default https://github.com/openthread/ot-br-posix/pull/724